### ISSUE TITLE
Monitor is not working

### ISSUE THREAD

**Issue by PaddeK** [neonious/lowsync#7]
I recently bought the neonious one and fiddled for a day within the web IDE.
Now with my first project in mind i would like to use lowsync with my prefered IDE.
Sync is working.. but i am unable to get any output from the MC to my PC.

I see following line
`--- User program's output: ---`
and then the command immediately exits.

Some general info:
- OS is Windows 7
- i tried console.log, console.warn and process.stdout.write
- code is syncing and running (tested with getting-started blink led example)
- tried cmd with administrator privileges

Let me know if you need more information etc.

**Comment by neoniousTR** [comment]
Thank you for reporting. We will try to find the root of the problem till Monday.

**Comment by neoniousTR** [comment]
Hello!
I tried out lowsync and everything is working perfectly. After --- User program's output: --- I see the output of the program. This works with console.log, console.warn and process.stdout.write.
Does lowsync completely exit (you see the command prompt again)? What's the Node.JS version?

As a next step we need to install a Windows system to test here...

**Comment by PaddeK** [comment]
Hi!

I am using NodeJS v10.16.3 and yes lowsync does completely exit and i can see the command prompt. But the program on the neonious is running indefinitely as expected.. blinking the LED.

Hier the sample Blinking code i use:
```
const gpio = require('gpio');

gpio.pins[gpio.LED_GREEN].setType(gpio.OUTPUT);

let val = 0;

setInterval(() => {
    console.log('HERE');

    gpio.pins[gpio.LED_GREEN].setValue(val = 1 - val);
}, 1000);
```

**Comment by neoniousTR** [comment]
OK. This is definitly a lowsync problem, because the microcontroller firmware is the same for everybody.

Trying it out on Windows ASAP. Today or tomorrow. Sorry for the long delay, we have to first handle the paying customers so we can pay our bills :-)

**Comment by neoniousTR** [comment]
As you can see in the screenshot, it works on my system:

![windows](https://user-images.githubusercontent.com/39573524/64628723-323e4980-d3f2-11e9-9460-3dfbca5e60fc.png)

There are some issues which we have to addres, however:
- I had to call `py -m pip install pyserial` instead of `pip install pyserial` which should be reflected in the text lowsync outputs
- I think it could be helpful to add to the docs that you can find the COM port with the `mode` command under Windows
- When flashing I had to press Enter sometimes
- It often says that it is "Testing the connection".. Working none-the-less. Maybe the timeouts are to small

BUT, as I said, the monitor output itself worked directly.

I am using a standard Windows 10.

What does py --version output on your system? On mine it says Python 3.8.0b4
Did you upgrade to lowsync 1.1.0. The newest version has updated dependencies. Maybe that fixes it

Thank you!

Thomas

**Comment by PaddeK** [comment]
I am stunned. Where in the docs its mentioned i need python? Right now i have no python installed.
As soon as i saw you updated lowsync to 1.1.0 i tried it with the same results. But if its as simple as installing python.. i will give it a try and report back.

**Comment by neoniousTR** [comment]
I am sure you had Python installed, overwise lowsync flash would not have worked.

Oh, ok, so http is not working? Well, then I know what to fix and why I did not notice the problem before. I am keeping this issue open till http is fixed. Not the highest priority though as most people will not switch to http.

**Comment by neoniousTR** [comment]
Also, I will fix the other problems with Windows:
- I had to call py -m pip install pyserial instead of pip install pyserial which should be reflected in the text lowsync outputs
- I think it could be helpful to add to the docs that you can find the COM port with the mode command under Windows
- When flashing I had to press Enter sometimes
- It often says that it is "Testing the connection".. Working none-the-less. Maybe the timeouts are to small

**Comment by neoniousTR** [comment]
Fixed 3 of 4 things with now published version 1.1.1.

Only thing missing: I think it could be helpful to add to the docs that you can find the COM port with the mode command under Windows

This is minor... Keeping the issue till done some day.

**Comment by neoniousTR** [comment]
Added a small info how to find the COM port here: https://neonious.com/lowjs/lowsync-doc/commands/commands/flash.html
